staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -2584,12 +2593,12 @@ func describeSecret(secret *corev1.Secret) (string, error) {
 		w.Write(LEVEL_0, "\nType:\t%s\n", secret.Type)
 
 		w.Write(LEVEL_0, "\nData\n====\n")
-		for k, v := range secret.Data {
+		for _, k := range sortStrings(secret.Data) {
```

Comment:
maybe you can just rely on maps.Keys, see: https://pkg.go.dev/maps#example-Keys

Reply:
@rikatz Great idea! Thanks!

Reply:
@rikatz Follow-up for using maps.Keys to simplify the code in other places as well 🤗

https://github.com/kubernetes/kubernetes/pull/130379

Reply:
@rikatz Added a follow-up for configmaps here: https://github.com/kubernetes/kubernetes/pull/131500